README.md

```diff
@@ -0,0 +1,117 @@
+
+>**Warning**
+>In this tutorial a customer system is built with no preexisting user base, which is effecively realized by having the loginInfo endpoint always return a 404 code when given a username. In case you have an existing user base, you just need to modify this endpoint to return other status codes as well, as described in our [docs](Link wenn online)
+
+## 1. File structure
+    .
+    ├── ...
+    ├── config                        
+    │   └── routes.yaml                 # Assigns paths to controller methods    
+    ├── src                             
+    │   ├── Controller                  
+    │   │   ├── BackendController.php   # Manages endpoints for backend
+    │   └── └── FrontendController.php  # Manages endpoints for frontend
+    ├── templates                     
+    │   ├── home.html.twig              # Home page which you only get to see if you are logged in
+    │   └── login.html.twig             # Login page which contains the Corbado web component
+    └── ...
+
+## 2. How to use
+>**Warning**
+>This sample code corresponds to our [web component integration tutorial](Link wenn online), please read it first in order to understand the flows and business logic!
+
+### 2.1. CNAME
+The only thing you need to create is a CNAME which points to `auth.corbado.com`. We will use `auth.your-company.com` in this tutorial. More info on what a CNAME is and why it is needed can be found in our [docs](https://docs.corbado.com/integrations/web-component#1.-define-cname).
+
+### 2.2. Setup
+
+#### 2.2.1. PHP Symfony
+To start the local development server your system requires [PHP](https://www.php.net/manual/en/install.php) and [Symfony](https://symfony.com/download):
+```
+brew install php
+brew install symfony-cli/tap/symfony-cli
+```
+
+You then need to install [Composer](https://getcomposer.org/download/):
+```
+curl -sS https://getcomposer.org/installer | php
+```
+With Composer you should be able to install all required packages for this project:
+```
+sudo -E env "PATH=$PATH" php composer.phar install
+```
+
+You should now be able to run this demo by typing `symfony server:start` into your console while being located in the root directory of this repository. If the setup was done correctly, the following messages should appear in your terminal:
+![image](https://user-images.githubusercontent.com/23581140/205909459-7ed3d679-b313-40d3-85be-1178b80a1594.png)
+To verify that your instance is running without errors enter `http://localhost:8000/ping` into your browser. If "pong" is displayed, you can continue with the next step.
+
+#### 2.2.2. Ngrok
+
+The endpoints of your local system have to be public so Corbado can send requests there. To make your local instance publicly availbale we use [ngrok](https://ngrok.com/download) which is a reverse proxy service. It assigns you a globally availably URL and forwards all incoming requests to your local instance. It can be installed using:
```

Comment:
Gib mal bitte noch die Alternative an, dass man sich da auch einfach ne .exe ziehen kann

Reply:
Meistens wirds so gemacht dass man nur die linux/mac commands angibt, weil die meisten darauf entwickeln und man in windows immer was runterladen und teilweise selber extracten und PATH setzen muss und das nicht einfach per befehl geht. Hätte oben ne warning rein dass die befehle halt nur für linux/mac sind, weiss nich obs unübersichtlich wird wenn man da anfängt bei den Befehlen per OS zu unterscheiden - würde unter windows eig eher raten ne wsl distribution aus dem microsoft store runterzuladen

Reply:
ok passt für mich